tags/pickMonster.tag

```diff
@@ -0,0 +1,92 @@
+<pickmonster>
+  <!-- the image select check is based of Birjitsinh-->
+  <!-- code found in https://bootsnipp.com/snippets/Zl6ql -->
+
+  <!-- HTML -->
+  <!-- Based of Birjitsinh code -->
+  <div class="container">
+    <h1 class="display-3 text-center">What monsters might help you on your journey?</h1>
+    <div class="row">
+      <form method="get">
+        <div class="form-group">
+          <div class="col-md-4" each={ monsterItem, i in myMonsters}>
+            <label class="btn btn-info">
+              <img src={ monsterItem.img } alt={ monsterItem.name } class="img-thumbnail img-check { check: monsterItem.pick }" onclick={ parent.toggle }>
+              <input type="checkbox" name={ monsterItem.name } id={ monsterItem.id } class="hidden">
+            </label>
+          </div>
+        </div>
+      </div>
+      <button type="button" class="btn btn-success" onclick={ selectMonster }>Next</button> <!-- need to add link to next page -->
```

Comment:
@mml2222 a helpful thing to put in the comment for something that needs to be done is "todo" - most IDEs will find all comments with "todo" in them and show them to you as to-do items!
(you don't need to change this, I just wanted to let you know for the future)

Reply:
Ok! Thank you!